README.md

```diff
@@ -29,127 +29,121 @@ module countBlock {
+The [module expressions](https://github.com/tc39/proposal-module-expressions) proposal introduces the expressions form of inline modules. Being expressions, they are inherently dynamic: they can be imported with `import()` or `new Worker()`, but not with static import statements.
 
-Module fragments lift this restriction: they can be imported statically if they appear at the top level of a module. This makes module fragments more useful for bundling than module blocks. See more context in [this FAQ](https://github.com/tc39/proposal-js-module-blocks#can-module-blocks-help-with-bundling).
+Module declarations lift this restriction: they can be imported statically if they appear at the top level of a module. This makes module declarations more useful for bundling than module expressions. See more context in [this FAQ](https://github.com/tc39/proposal-module-expressions#can-module-blocks-help-with-bundling).
 
-If module fragments appear somewhere else (within an `eval`, in a legacy script, in an inline module, nested inside of a function, …), they act similar to a module block that is assigned to a variable:
-
-```js
-const mod = module { ... };
-module mod { ... }
-new Worker(mod);
-```
+We are developing these two features as separate proposals because module declarations present additional complexity around static import statements that module expressions don't need, and module expressions have their own [motivations](https://github.com/tc39/proposal-module-expressions#problem-space) independent from module declarations. Module declarations inherit different design decisions from module expressions, so the advancement of this proposal in its current shape depends on the evolution of module expressions.
 
 ### Does this proposal work with import maps?
 
-[Import maps](https://github.com/WICG/import-maps) can be used in conjunction with module fragments, in that the import map can redirect bare specifiers to be found in module fragments, rather than independent fetches. Mechanically: The lookup in the import map (which is done as part of "resolving a module specifier") precedes the interpretation of fragments in the module specifier (which are treated as part of the module map key). (TODO: add example of use together.)
+[Import maps](https://github.com/WICG/import-maps) can be used in conjunction with module declarations, in that the import map can redirect bare specifiers to be found in module declarations, rather than independent fetches. Mechanically: The lookup in the import map (which is done as part of "resolving a module specifier") precedes the interpretation of declarations in the module specifier (which are treated as part of the module map key). (TODO: add example of use together.)
```

Comment:
Note, this part doesn't really apply anymore without import map extensions; we should track that in an issue.

Reply:
I commented this section out for now, and renamed https://github.com/tc39/proposal-module-fragments/issues/10 to include "import maps" in the title (since it's effectively what that issue is about).